### PR TITLE
Release v1.2.15

### PR DESCRIPTION
**Pull request by oriAdler**
- fix: improve error messages for deleting oidc-config
- fix: adding some validations to bucket name
- fix: allow empty label match editing ingress interactive
- aws: Ensure ARNs have the correct partition
- Attach three policies to the installer role - managed policies
- to fix empty DNS domain when DNS not ready: [SDA-7418](https://issues.redhat.com//browse/SDA-7418)
- fix: add a few more validations to bucket/folder name
- Add a `AWS managed` column to list role commands
- Add etcd encyprtion kms arn support

### CODE REVIEW

File: hack/commit-release.sh [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 git fetch --tags
 tagcommit=$(git rev-list --tags --max-count=1)
-current=$(git describe --tags $tagcommit | sed 's/v//')
+#current=$(git describe --tags $tagcommit | sed 's/v//')
+current="1.2.14"
```
Comment (gdbranco) [inline hack/commit-release.sh:23]: This shouldn't be needed
You usually fix it by `git fetch upstream` to get latest tags locally

Comment (oriAdler) [reply hack/commit-release.sh]: Good catch!
Fixed, thanks.

**Pull request merged**